stabilizer/src/main/java/com/hazelcast/stabilizer/worker/selector/OperationSelectorBuilder.java

```diff
@@ -0,0 +1,86 @@
+package com.hazelcast.stabilizer.worker.selector;
+
+import org.apache.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.format;
+
+/**
+ * Builder to create an {@link OperationSelector} instance. Each operations is represented as an enum item
+ * and is registered with its probability percentage (0 to 100) via the {@link #addOperation(Enum, int)} method.
+ * <p/>
+ *
+ * @param <T> enum of operations
+ */
+public class OperationSelectorBuilder<T extends Enum<T>> {
```

Comment:
I don't question that we should create an instance per worker and I will update the doc to recommend this. Nevertheless I really have in mind that `Random` is be thread-safe. I looked it up and it is in the docs since Java 7: http://docs.oracle.com/javase/7/docs/api/java/util/Random.html

So yes, maybe it will fail for all Hazelcast related projects on Java 6, so we should not relay on it.